### PR TITLE
Es6 projects common modules identity cookierefresh

### PR DESCRIPTION
**Pull request by nicl**
Converts cookie refresh module to ES6, including tests.

(p.s. stole this from @MatthewJWalls sorry - had some spare time but didn't look like you'd started on it.)

### CODE REVIEW

File: static/src/javascripts/projects/common/modules/identity/cookierefresh.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    isUserLoggedIn,
+    getUserFromApiWithRefreshedCookie,
+} from 'common/modules/identity/api';
+import { local as localStorage } from 'lib/storage';
+
+const shouldRefreshCookie = (
+    lastRefresh: ?number,
+    currentTime: number
+): boolean => {
+    const days30k = 1000 * 86400 * 30; // (as seconds)
```
Comment (nicl) [inline static/src/javascripts/projects/common/modules/identity/cookierefresh.js]: Not sure what the significance of this is. I'd prefer to go with, e.g. `86400 * 365 * 5` (5 years approx) but it doesn't really matter.

Comment (SiAdcock) [reply static/src/javascripts/projects/common/modules/identity/cookierefresh.js]: Probably no special significance. It was implemented 2.5 years ago without explanation (#9191) 😄

**Review by SiAdcock — APPROVED** [review]
In the words of Shakin' Stevens: lovely stuff

**Review by gu-stav — APPROVED** [review]
Nais!

**Review by MatthewJWalls — APPROVED** [review]

**Pull request merged**